e2e/test/scenarios/dashboard-cards/click-behavior.cy.spec.js

```diff
@@ -586,6 +608,13 @@ describe("scenarios > dashboard > dashboard cards > click behavior", () => {
         expect(card.dataset_query.query).to.deep.equal(TARGET_QUESTION.query);
       });
 
+      cy.log("Should navigate to question using router (metabase#33379)");
```

Comment:
Piggyback assertions that test the navigation have been done using router.

If you check out `master` and run this test. The test should fail.